docs/developer/add-new-source.md

```diff
@@ -1,6 +1,14 @@
 # How to add a new source to the Mondo Ingest
 
-1. Add a new import and component to [src/ontology/mondo-ingest-odk.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/mondo-ingest-odk.yaml).
-2. Add a new metadata file to [src/ontology/metadata](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/metadata). It is important we try and document as much about the source as we can.
-3. Update the Website Table of Contents [mkdocs.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/mkdocs.yaml)
+## 1. `mondo-ingest-odk.yaml`
+### 1.1. Add source info
+Add a new filename and URL to the `products` > `components` section in [src/ontology/mondo-ingest-odk.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/mondo-ingest-odk.yaml).
 
+### 1.2. `update_repo`
+From `src/ontology`, run `sh run.sh make update_repo`.
+
+## 2. `metadata/*.yaml`
+Add a new metadata file to [src/ontology/metadata](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/metadata). It is important we try and document as much about the source as we can.
+
+## 3. `mkdocs.yaml`
```

Comment:
@matentzn Do you think something is missing here? I noticed that it looks like I manually needed to add links to `gard.md` in several places. It doesn't look like `mkdocs build` is what I want to run. Perhaps we need to write something here about manually making some other additions? Such as:

1. `sources/ONTOLOGY.md`
2. Adding ONTOLOGY to `sources.md`
3. What about `metrics/ONTOLOGY.md` as well as adding a link to that file in `mkdocs.yaml` as well?

Reply:
Unfortunately ues, mkdocs.yaml is entirely automatically managed. So all of these need to be added.

Reply:
Np. I added instructions for adding these. I see now that there are some goals that help with these:
`../../docs/sources/%.md`
`../../docs/metrics/%.md`